lib/task/search/arPopulateTask.class.php

```diff
@@ -48,6 +50,6 @@ public function execute($arguments = array(), $options = array())
 
     new sfDatabaseManager($this->configuration);
 
-    QubitSearch::getInstance()->populate();
+    QubitSearch::getInstance()->populate($options);
```

Comment:
Can we explicitly pass here the options we want here to avoid undefined behaviour? This reminds me to the use of `locals()` in Django views, that we try to avoid.